src/patch.cpp

```diff
@@ -1801,8 +1797,6 @@ ScratchSharedPtr Patch::Compile()
             }
             return nullptr;
         }
-
-        std::unreachable();
```

Comment:
Was std::unreachable used incorrectly here?  Or does msvc not support it or such?

Reply:
Ah, I saw the commit message that explains it :3